unified-installer/astra-unified-installer.sh

```diff
@@ -726,7 +726,8 @@ loginfo() {
 }
 
 logwarn() {
-    log_at_level $__WARN "WARNING: $1"
+    local -r warning="${1#WARNING: }"
+    log_at_level $__WARN "WARNING: $warning"
```

Comment:
This is just to prevent having some kubectl warnings say `WARNING: WARNING: blablabla`.